modules/sdk-core/src/bitgo/utils/tss/baseTSSUtils.ts

```diff
@@ -108,6 +109,20 @@ export default class BaseTssUtils<KeyShare> extends MpcUtils implements ITssUtil
     throw new Error('Method not implemented.');
   }
 
+  /**
+   * Converts user and backup MPCv1 SigningMaterial to RetrofitData needed by MPCv2 DKG.
+   *
+   * @param {Object} params - MPCv1 decrypted signing material for user and backup as a json.stringify string and bitgo's Big Si.
+   * @returns {Promise<{ mpcv2UserKeyShare: DklsTypes.RetrofitData; mpcv2BakcupKeyShare: DklsTypes.RetrofitData }>} - the retrofit data needed to start an MPCv2 DKG session.
+   */
+  getMpcV2RetrofitDataFromMpcV1Keys(params: {
+    mpcv1UserKeyShare: string;
+    mpcv1BackupKeyShare: string;
+    bitgoBigSi: string;
+  }): Promise<{ mpcv2UserKeyShare: DklsTypes.RetrofitData; mpcv2BakcupKeyShare: DklsTypes.RetrofitData }> {
+    throw new Error('Method not implemented.');
```

Comment:
is this supposed to be not implemented, im assuming yes, but just curious

Reply:
yeah the pattern for this class is meant to be abstract in theory.

Reply:
I think you can add it a isolated function if you dont need anything from the class or the super

Reply:
it felt like this migration function belonged under mpcv2 utilities so i put it there under the mpcv2 utility class

Reply:
Since this is not a common utility between all the MPC utils classes, you can either move it into an isolated function or remove it from the base class, you need to instantiate the MPCv2 util class anyways to use it.

Reply:
makes sense, will move this function

Reply:
removed from base class